index.html

```diff
@@ -2141,7 +2145,7 @@ <h2 id="intentionally-causing-an-exception"><a
 <ul>
 <li>The pseudoinstruction <span
 id="insn-unimp"><em><code>unimp</code></em></span> has the same encoding
-as <code>csrrw zero, cycle, zero</code>, and it is the canonical RV32I
+as <code>csrrw zero,   cycle, zero</code>, and it is the canonical RV32I
```

Comment:
I don't know why pandoc added spaces here for me but not for you. Let me remove them manually.